### DISCUSSION TITLE
Cypress is asserting in the original HTML source of the page (`view-source`) instead of the DOM, is that ok?

### DISCUSSION THREAD

**Discussion by melibe23** [cypress-io/cypress#15114] [Questions and Help]
Current tag in DOM Chrome:

<meta content="null" name="robots">
My test:

it(`Check "meta[name="robots"]`, function() {
cy.get(`meta[name="robots"]`)
              .should(`exist`) 
              .and("have.attr", "content", "null"); 
});
Outcome:
image
![image](https://user-images.githubusercontent.com/32499737/108089217-a403d380-7079-11eb-980a-9a7b23eccd83.png)

Expected (or maybe I am wrong) > attr CONTENT should exist and its value should be the string "null".

This is happening because Cypress is asserting in the original HTML source of the page (view-source) instead of the DOM, is that ok?

Data:
Cypress Version 6.4.0
Chrome Version 88.0.4324.150 (Official Build) (x86_64)
MacOS Catalina 10.15.7 (19H2)

**Comment by bahmutov** [comment]
What is the markup for your meta robots element?
